components/atoms/UserCard/user-card.tsx

```diff
@@ -32,9 +32,9 @@ const UserCard = ({ username, name, meta, loading }: UserCardProps) => {
               src={avatarUrl}
               alt={`${username}'s avatar image`}
             />
-            <div>
+            <div className="text-center">
               <h3 className="text-base text-center">{name}</h3>
```

Comment:
```suggestion
              <h3 className="text-base">{name}</h3>
```
Remove the unused `text-center` utility class, alignment will be inherited from the containing `div`